docs/faq.rst

```diff
@@ -0,0 +1,3 @@
+=====================================================
+Frequently Asked Questions
+=====================================================
```

Comment:
A couple of template questions:
* How does this SDK work?
* What is PMA?
* I keep getting an Exception - what is the problem?